src/pages/Catalogue.js

```diff
@@ -17,9 +17,6 @@ import { Helmet } from 'react-helmet';
 
```

Comment:
If I make edits to an existing import, usually I will keep it as single quotes but since the project settings are double quotes if I make any major edits to the imports I will change them to double quotes. I didn't make any edits to those lines though so they are kept as single quotes.

Reply:
Are the automatic linting settings not working for you?